docs/recipes/upgrading/v4-to-v5.md

```diff
@@ -0,0 +1,257 @@
+	nuget Fable.Elmish ~> 4
+	nuget Fable.Elmish.React ~> 4
+	nuget Fable.Elmish.Debugger ~> 4
+	nuget Fable.Elmish.HMR ~> 7
+	nuget Fable.Remoting.Client ~> 7
+	nuget Feliz ~> 2
+	
+    //Build
+	nuget Fake.Core.Target ~> 5
+	nuget Fake.IO.FileSystem ~> 5
+	nuget Farmer ~> 1
+
+    //Testing
+	nuget Fable.Mocha ~> 2
+	nuget Expecto ~> 9
+	```
+	
+	- Run `dotnet paket install`
+	- Remove `Fable.React` and `Feliz.Bulma` from `src/Client/paket.references` (moved here according to @Freymaurer feedback)
+    - Run `dotnet paket update Fable.Remoting.Client`, needs to be done separately because version changes is a minor one and not major so I can't force it using only the `paket.dependencies` changes
+
+1. Remove all the webpack related dependencies:
+
+	```bash
+	npm uninstall copy-webpack-plugin \
+	    css-loader \
+	    file-loader \
+	    html-webpack-plugin \
+	    mini-css-extract-plugin \
+	    sass-loader \
+	    source-map-loader \
+	    style-loader \
+	    webpack \
+	    webpack-cli \
+	    webpack-dev-server
+	```
+	
+1. Update the left over NPM dependencies:
+
+	- You can run to upgrade all at once `npx npm-check-updates -u` or if you prefer to be more conservative you can use `npx npm-check-updates -i`.
+	
+	If you use the second option, make sure to select `react`, `react-dom`, `sass`.
+	
+1. Install the new NPM dependencies
+
+	`npm install -D vite @vitejs/plugin-react tailwindcss postcss autoprefixer @types/node`
+	
+1. Delete `webpack.config.js`
+
+1. Create the file `src/Client/vite.config.mts` with the following content
+	
+	```ts
+	import { defineConfig } from "vite";
+	import react from "@vitejs/plugin-react";
+	
+	const proxyPort = process.env.SERVER_PROXY_PORT || "5000";
+	const proxyTarget = "http://localhost:" + proxyPort;
+	
+	// https://vitejs.dev/config/
+	export default defineConfig({
+	    plugins: [react()],
+	    build: {
+	        outDir: "../../deploy/public",
+	    },
+	    server: {
+	        port: 8080,
+	        proxy: {
+	            // redirect requests that start with /api/ to the server on port 5000
+	            "/api/": {
+	                target: proxyTarget,
+	                changeOrigin: true,
+	            }
+	        }
+	    }
+	});
+	```
+
+1. Overwrite `src/Client/index.html` with the following content
+	
+	```html
+	<!doctype html>
+	<html>
+	<head>
+	    <title>SAFE Template</title>
+	    <meta charset="utf-8">
+	    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	    <link rel="icon" type="image/png" href="/favicon.png"/>
+	    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css">
+	</head>
+	<body>
+	    <div id="elmish-app"></div>
+	    <!-- Polyfill for remotedev -->
+	    <script type="text/javascript">
+	        var global = global || window;
+	    </script>
+	    <script type="module" src="/output/App.js"></script>
+	</body>
+	</html>
+	```
+	
+1. Run `npx tailwindcss init -p` in `src/Client`
```

Comment:
Should the tailwind and postcss config files be added to proj files? They are in the template

Reply:
See https://github.com/SAFE-Stack/SAFE-template/blob/master/Content/default/src/Client/Client.fsproj

Reply:
We need to add this

Reply:
This is unresolved.

Reply:
Not going to worry about this for now. Will get clarity from Maxime and can update later.